examples/hello-world/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
-    println!("Hello, world!");
+    println!("Hello, World!");
```

Comment:
😄

Reply:
Ha, in a hindsight that's a too petty of a change - I've always thought the upper-cased variant is correct, but now I see `cargo new` yields `Hello, world!`, so probably best to stick to it 😄